poc/vdaf_prio3.py

```diff
@@ -323,14 +348,14 @@ def joint_rand_seed(Prio3, k_joint_rand_parts):
         )
 
     @classmethod
-    def joint_rand(Prio3, k_joint_rand_seed):
+    def joint_rands(Prio3, k_joint_rand_seed):
```

Comment:
As @divergentdave suggested, let's incorporate `Prio3.PROOFS` into the transcript for deriving the joint rands.

Reply:
If Prio3’s derivation of domain separation tag already includes algorithm ID, is it sufficient to make algorithm ID specify a distinct combination of `(Field, PROOFS)`, per thread [here](https://github.com/cfrg/draft-irtf-cfrg-vdaf/pull/298#discussion_r1344864217)?

Reply:
That's one possibility, but it somewhat depends on how "configurable" we need the value of `PROOFS` to be. Ideally we can fix it for a given choice of circuit and field.

Reply:
I think for now it makes sense to include it in the transcript.

Reply:
>I think for now it makes sense to include it in the transcript.

Just to clarify. And do we want to use binder string?

Reply:
Yup, at the step where we derive the joint rands from the `k_joint_rand_seed`.

Reply:
If we treat (`PROOFS, Field`) as parameter for the underlying algorithm, i.e. same as `SHARES`, it makes senses to add them into joint randomness derivation. 
1. If we do so, maybe we can allocate one algorithm ID/code point for Prio3SumVec variant with multiproofs? 
2. Shall we add field choice into joint randomness derivation too (binder) ?

Reply:
> If we treat (`PROOFS, Field`) as parameter for the underlying algorithm, i.e. same as `SHARES`, it makes senses to add them into joint randomness derivation.
> 
>     1. If we do so, maybe we can allocate one algorithm ID/code point for Prio3SumVec variant with multiproofs?

I don't think we should actually allocate a codepoint until we know more about robustness. In the meantime, one is free to use any of the unreserved codepoints however they wish.


>     2. Shall we add field choice into joint randomness derivation too (binder) ?

We maybe don't need to unless we plan to use more than one field. Isn't `Field64` the only one that's needed?

Reply:
OK. Only `Field64` for now.